veg2hab/vegkartering.py

```diff
@@ -1171,6 +1211,60 @@ def from_editable_vegtypes(cls, gdf: gpd.GeoDataFrame) -> Self:
         )
         return cls(gdf)
 
+    @staticmethod
+    def combineer_karteringen(karteringen: List[Self]) -> Self:
+        """
+        Accepteert een lijst met karteringen. Deze worden gecombineerd door ze
+        op de volgorde in de lijst over elkaar heen te leggen.
+
+        Data binnen de vlakken blijft gelijk, buiten dat er een nieuwe ElmID
+        wordt gegenereerd en de oppervlakten (gdf.Area) opnieuw berekend worden
+
+        Input:   [kart1, kart2, kart3]
+                    +-------+
+        kart1:      |   A   |
+                    +-------+
+                        +-------+
+        kart2:          |   B   |
+                        +-------+
+                            +-------+
+        kart3:              |   C   |
+                            +-------+
+
+                    +---+---+-------+
+        Output:     | A | B |   C   |
+                    +---+---+-------+
+        """
+        assert (
+            len(karteringen) > 1
+        ), "Er moeten minstens 2 karteringen zijn om te combineren"
+
+        for kartering in karteringen:
+            assert isinstance(
+                kartering, Kartering
+            ), "Alle elementen in karteringen moeten Karteringen zijn"
+            assert all(
+                col in kartering.gdf.columns for col in kartering.VEGTYPE_COLS
+            ) and not all(
+                col in kartering.gdf.columns for col in kartering.HABTYPE_COLS
+            ), "combineer_karteringen moet als stap 2 uitgevoerd worden"
+
+        result = karteringen[0].gdf
+        for kartering in karteringen[1:]:
```

Comment:
Jij verwacht volgens mij kartering in volgerde van lage prio naar hoge prio. Volgens mij communiceert onze UI het in de tegenovergesteld richting. Dus moeten we even ergens reversen.

Reply:
Ik gooi wel even een .reverse() in `run_2_stack_vegkartering`